helm/zenko/values.yaml

```diff
@@ -43,15 +48,7 @@ prometheus:
   pushgateway:
     enabled: false
 
-global:
-  # These setting get passed to the mongodb-replicaset helm
-  # Along with the Cloud Server front end for proper communication
-  mongodbBackend: 
-    enabled: true
-    replicaSet: rs0
-    # The 'rs0' is the default mongo replicaset name
-    replicas: 5
-    # The number of replicas to have in the cluster
-    port: 27107
-    # The default port number used by mongo
-
+mongodb-replicaset:
```

Comment:
Shouldn't we keep some comments to describe what all these values mean?

Reply:
Not sure, TBH. We're not inventing new configuration values, simply overriding whatever is passed by default in the mongodb-replicaset chart values.yaml, where they are (or should be) documented. Duplicating this here may make things go out of sync easily.